### DISCUSSION TITLE
Inquiry into how example json bodies on POST requests are generated.

### DISCUSSION THREAD

**Discussion by eliaspinno-alltrue** [fastapi/fastapi#11456] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
# Some example pydantic classes

class ResourceToAdd(BaseModel):
    resource_type: str
    resource_data: dict
    technology_types: list[str]
    cloud_provider_account_id: str | None = None


class AddMultipleResourceInstances(BaseModel):
    customer_id: uuid.UUID | None = None
    resources: list[ResourceToAdd]

# after running uvicorn app.api.main:app --host 0.0.0.0 --port 8000 --reload
# The json yielded on the front end as an example when running the command and looking at that endpoint on the generated
# /docs UI. This showed up as an example for a POST request which had response_model=AddMultipleResourceInstances


"""
{
  "customer_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6", # Not a real customer ID, don't worry
  "resources": [
    {
      "resource_type": "string",
      "resource_data": {},
      "technology_types": [
        "string"
      ],
      "cloud_provider_account_id": "string"
    }
  ]
}
"""
```

### Description

I notice that the auto-generated docs for my APIs have these nice and nifty example responses for all of the APIs that accept a JSON body. I'd like to use the exact logic that generated these very nice example JSONs elsewhere in my application to represent some example JSONs that could create other `BaseModel` objects. But try as I might, I wasn't able to find where these JSONs were generated in the source code to this repo.

I browsed the pydnatic docs and forums a decent amount in advance to ask this question: I'm aware of this thread https://github.com/pydantic/pydantic/issues/889 which proposes this solution at the bottom of it:

`jsonref.replace_refs(SomeModel.model_json_schema())`

But the problem with that solution is it still leaves me with a lot of trailing "properties" dictionaries and other extraneous data, etc. I just want the same dictionary that was generated and displayed on the front end via Python code. I'm aware I could probably hack a solution together on my own locally after replacing the refs, but would prefer to use an existing, well-tested tool. 

### Operating System

Linux

### Operating System Details

Ubuntu

### FastAPI Version

0.110.0

### Pydantic Version

2.6.4

### Python Version

3.11

### Additional Context

Image of what I am talking about:

![image](https://github.com/tiangolo/fastapi/assets/148122261/198d00f1-5952-4bf2-ba76-66bd0020dc19)

**Comment by YuriiMotov** [comment, marked as answer]
I think these examples are generated by Swagger, not FastAPI. You can get `http://127.0.0.1:8000/openapi.json` file and make sure of this (there are no any of these auto-generated examples there).
FastAPI generates `openapi.json` file, and then Swagger requests it and creates documentation using this openapi schema.
So, this code is in Swagger's JavaScript files.
